### ISSUE TITLE
SSL connection could not be established

### ISSUE THREAD

**Issue by waterboy550** [linvi/tweetinvi#1157]
I've been running the tweetinvi library for a long time, recently I've been getting a ton of the errors below when trying to read a list I have setup.

The code I'm using is below.
userTimeline = await userClient.Lists.GetTweetsFromListAsync(1360674149816037376);

The error I'm getting is below, is this something with my code... or is Twitter simply being really flaky as of late? This error happens randomly from time to time no rhyme of reason, sometimes it doesn't error out and sometimes it does.

The SSL connection could not be established, see inner exception.    at System.Net.Http.ConnectHelper.EstablishSslConnectionAsyncCore(Stream stream, SslClientAuthenticationOptions sslOptions, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)

**Comment by cblaze22** [comment]
I am getting

```
Message :Reason : The request was aborted: Could not create SSL/TLS secure channel.
Code : -1
Date : 3/17/2022 4:09:52 PM -04:00
URL : https://api.twitter.com/1.1/statuses/destroy/44444.json?tweet_mode=extended
Twitter documentation description : 

Source :Tweetinvi.WebLogic
Stack Trace :   at Tweetinvi.WebLogic.WebRequestExecutor.<ExecuteTwitterQuerySafelyAsync>d__7`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Tweetinvi.WebLogic.TwitterRequestHandler.<ExecuteQueryAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Tweetinvi.WebLogic.TwitterRequestHandler.<ExecuteQueryAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Tweetinvi.Credentials.TwitterAccessor.<ExecuteRequestAsync>d__4`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Tweetinvi.Client.BaseRequester.<ExecuteRequestAsync>d__8`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Tweetinvi.Client.TweetsClient.<DestroyTweetAsync>d__16.MoveNext()
TargetSite :Void MoveNext()
```

**Comment by waterboy550** [comment]
Weird thing, I was running my program on a Windows 2012 R2 server and the errors would happen there. When I would run on my coding computer, Windows 11 I never had issues. I moved the program to run on a Windows 10 desktop that acts as an "application server" for a few things. No issues since I moved the program over. No code changes, nothing was changed other than the computer it was running on. Must be a security thing with 2012 R2 that was applied via an update. It was running fine for a year previously. Tried with all anti virus off and firewall off as well, still got errors.
